test/Reachability/unit_solve_hybrid_bouncing_ball.jl

```diff
@@ -14,7 +14,7 @@ A = [0.0 1.0; 0.0 0.0];
 B = reshape([0.0, -1.0], (2, 1));
 X = HalfSpace([-1.0, 0.0], 0.0); # x >= 0
 U = Singleton([1.0]);
-m = [ConstrainedLinearControlContinuousSystem(A, eye(size(B, 1)), X, B*U)];
+m = [ConstrainedLinearControlContinuousSystem(A, Matrix{eltype(A)}(I, size(B, 1), size(B, 1)), X, B*U)];
```

Comment:
Why `size(B, 1), size(B, 1)`? `B` is not quadratic!

Reply:
`eye(size(B, 1))` produces a square matrix  (shortcut of `eye(size(B, 1), size(B, 1))`).

this says `Ax + v  = Ax + Id_{nxn} v` and `v` is in `B*U`, `u`  1-dim but `v` must be `n`-dim.